src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -27,17 +27,16 @@
  */
 
 #include "squid.h"
+#include "debug/Stream.h"
 #include "helper/protocol_defines.h"
+#include "sbuf/SBuf.h"
+#include "sbuf/Stream.h"
 
-#include <cstdlib>
-#include <cstring>
+static const int DEBUG_SECTION=1;
```

Comment:
Please use a more specific section (e.g., the existing section 82 "External ACL").

```suggestion
static const auto DEBUG_SECTION = 82; // External ACL
```

Also, for consistency with TcpLogger, consider adding a `MY_` prefix:

```suggestion
static const auto MY_DEBUG_SECTION = 82; // External ACL
```

Eventually, scripts/source-maintenance.sh should start looking for declarations like this (in addition to `DEBUG: section` comments it is processing right now).

Please also move this declaration lower, after `#include` statements.

Reply:
ok, done